### PR TITLE
Hw17 MaksimKez

### PR DESCRIPTION
**Pull request by MaksimKez**
Задание сделано, но есть недочет. После завершения работы новые (измененные) настройки не отображаются в appsettings.Development.json (как и в appsettings.json). Как это решить не знаю :(

### CODE REVIEW

**Comment by dkobzin** [comment]
Добрый день! Убедительная просьба - не мержить самостоятельно PR в main.

File: MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using Newtonsoft.Json;
+
+namespace hw16.Models
+{
+    [Serializable]
```
Comment (dkobzin) [inline MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs:5]: аттрибуты тут не нужны

File: MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    public class MeetingSettings
+    {
+        [JsonProperty("MaxPeople")]
```
Comment (dkobzin) [inline MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs:8]: Эти то же

File: MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        public int MaxPeople { get; set; }
+
+        [JsonProperty("MaxTime")]
```
Comment (dkobzin) [inline MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs:11]: И этот не нужен

File: MaksimKez/HW17/hw16/hw16/Program.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+            builder.Services.AddControllersWithViews();
+            builder.Services.AddTransient<IGetRoomSettings, MeetingSettingsService>();
```
Comment (dkobzin) [inline MaksimKez/HW17/hw16/hw16/Program.cs:14]: AddTransient смените на AddScoped. Имена интрефейса и сервиса должны совпадать в общем. Интерфейс должен иметь префикс I

File: MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        void IGetRoomSettings.SaveSettings(MeetingSettings meetingSettings)
+        {
+            // изменить в appsettings.Development.json по ключу значения MeetingSettings
+            //var jsonStr = File.ReadAllText("appsettings.Development.json");
+            //var jsonSettingsToChange = System.Text.Json.JsonSerializer.Deserialize<MeetingSettings>(jsonStr);
+            //jsonSettingsToChange.MaxPeople = meetingSettings.MaxPeople;
+            //jsonSettingsToChange.MaxTime = meetingSettings.MaxTime;
+            //var
```
Comment (dkobzin) [inline MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs:30]: Уберите пожалуйста закомментированный код.

File: MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+        void IGetRoomSettings.SaveSettings(MeetingSettings meetingSettings)
+        {
+            // изменить в appsettings.Development.json по ключу значения MeetingSettings
+            //var jsonStr = File.ReadAllText("appsettings.Development.json");
+            //var jsonSettingsToChange = System.Text.Json.JsonSerializer.Deserialize<MeetingSettings>(jsonStr);
+            //jsonSettingsToChange.MaxPeople = meetingSettings.MaxPeople;
+            //jsonSettingsToChange.MaxTime = meetingSettings.MaxTime;
+            //var 
+            _configuration["MeetingRoomSettings:MaxPeople"] = meetingSettings.MaxPeople.ToString();
```
Comment (dkobzin) [inline MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs:31]: Тут что то будети подобное  configuration["Meetings"] = JsonSerializer.Serialize(meetings);

**Review by dkobzin — CHANGES_REQUESTED** [review]

**Comment by dkobzin** [comment]
Добрый день!

Посмотрите пожалуйста комментарии.

С уважением,

Дмитрий Кобзин
________________________________
From: MaksimKez ***@***.***>
Sent: Thursday, April 4, 2024 9:43 AM
To: dkobzin/tms-net15-onl ***@***.***>
Cc: Dmitry Kobzin ***@***.***>; Review requested ***@***.***>
Subject: [dkobzin/tms-net15-onl] Hw17 MaksimKez (PR #78)


Задание сделано, но есть недочет. После завершения работы новые (измененные) настройки не отображаются в appsettings.Development.json (как и в appsettings.json). Как это решить не знаю :(

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/dkobzin/tms-net15-onl/pull/78

Commit Summary

  *   c7defcd<https://github.com/dkobzin/tms-net15-onl/pull/78/commits/c7defcd2d6f81c2a7a0efcdf4e4a6319cf984d62> made partially
  *   cf0a511<https://github.com/dkobzin/tms-net15-onl/pull/78/commits/cf0a5113345b60fab7f353baead8032e2015d65a> made without saving

File Changes

(82 files<https://github.com/dkobzin/tms-net15-onl/pull/78/files>)

  *   M .gitignore<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947> (3)
  *   A MaksimKez/HW17/hw16/hw16.sln<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-6d65c0128c0ed3c62e42c77038a13bcf3621d655f452a6e3d833aba8458dd0bb> (25)
  *   A MaksimKez/HW17/hw16/hw16/Controllers/HomeController.cs<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-446b57dfe3a976ad05203a8c599a8bc9b0316f29a1729fc7e123500bdbbfe88a> (61)
  *   A MaksimKez/HW17/hw16/hw16/Models/ErrorViewModel.cs<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-2509a2a9672d1250de10e49d8174574c4008e1b97a23a7c66a6793664b38d1e2> (9)
  *   A MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-33ae5fa890c726a92bc2d163b3d911f432e7e08fd0c1aefbeaa4faf80561b0cf> (14)
  *   A MaksimKez/HW17/hw16/hw16/Program.cs<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-f12b177e11e7d33a6bc4d1c375a2871b378e07cb4e397f3514b4bb932ab08a65> (39)
  *   A MaksimKez/HW17/hw16/hw16/Properties/launchSettings.json<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-88a56613e318c861f362f738d59d12f78dcf8891e41672bccb4db17ced348d5f> (38)
  *   A MaksimKez/HW17/hw16/hw16/Services/IGetRoomSettings.cs<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-cd1fb7ec16a0c9d8228707263edc4d5bda982910fdbc529045503af3c7d51653> (10)
  *   A MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-d7842fe6f2c391ce0c9b945c2c414d97305b4b17570c63192c36e152cac492c0> (51)
  *   A MaksimKez/HW17/hw16/hw16/Views/Home/Index.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-434f7807a21596a124e54f84e65a2e98909f0d9e078d3193e1a66e0ab32b26c1> (25)
  *   A MaksimKez/HW17/hw16/hw16/Views/Home/MyTest.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-24e3e7b9d64606bddeca746e3d77584ace1f16c6aaf38a0aca1b0884b87eb18d> (12)
  *   A MaksimKez/HW17/hw16/hw16/Views/Home/Privacy.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-996a2dac6980e9198869843cf9a8dc3984e47a5238a6d9545e0aff889b7b996b> (6)
  *   A MaksimKez/HW17/hw16/hw16/Views/Shared/Error.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-ae16272fe85dde9f4bb632cc8c4ff1a9f0e857bb684ca0e0c4ebfdab1e4f8b4e> (25)
  *   A MaksimKez/HW17/hw16/hw16/Views/Shared/_Layout.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-7a7497ea532074ed991647724264d639b51a3b2943cc1fe6b165c79c9e6823e2> (49)
  *   A MaksimKez/HW17/hw16/hw16/Views/Shared/_Layout.cshtml.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-49d4e4aa61a74e6456fb7dd92ca56a2fb8e60bf3181f5a3ff68a8ff08fb6f39a> (48)
  *   A MaksimKez/HW17/hw16/hw16/Views/Shared/_ValidationScriptsPartial.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-85523d1f6c1cef9a9d96192fc474f29192dd4140c2a12b4f65dd0180498f02de> (2)
  *   A MaksimKez/HW17/hw16/hw16/Views/_ViewImports.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-89cb3d0d4bbe68e64e39338f5db32464d44dbbd45726704ff444d1288c6383fe> (3)
  *   A MaksimKez/HW17/hw16/hw16/Views/_ViewStart.cshtml<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-94550e9f7ec7477a2e86db3adaccfdea739e44abfd6203333d0420f2bdcfda8a> (3)
  *   A MaksimKez/HW17/hw16/hw16/appsettings.Development.json<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-89c47034844a53a2db430d13e199bc4a34bee94f6ff1960dc314bd4ed88d5185> (14)
  *   A MaksimKez/HW17/hw16/hw16/appsettings.json<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-476efd71e0b3c859d9b1975c1e864c7657673a0b8ee82ed40da3701e5835ff02> (14)
  *   A MaksimKez/HW17/hw16/hw16/hw16.csproj<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-f97d0bc0a5d4b4f2a0398a3493058325d760dbd71d7ffb326fc977be294676ff> (13)
  *   A MaksimKez/HW17/hw16/hw16/hw16.csproj.user<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-8e56c93901dec28bb935bb5d383ee249490ded80cb76b86f4dc13236e9e9101f> (11)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/css/site.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-9e25520c521a58cbf985ba17093df6037f0f67a4a74015d89d713d41005fb7f9> (22)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/favicon.ico<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-7348f1cc48a2072dfb66ddcfe4509864ebbdf291c6733804e716c50199cba63f> (0)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/js/site.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-311ba521e17a66fb71405aa8b5bc11815796676ad8b9484dbd172e4b448d8ca2> (4)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/LICENSE<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-b661bae74a089506b42cfd3e5f4d0508bb86454604af26627ad01b4a74582655> (22)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-ebf1ffec5bd06006fd6c34b1abc8a831cf9d51981041c826a70bff4e9243cd1e> (4997)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-4c50138ddd508a47444cd6273882f499667ce12df03653553ab17e82756f4a29> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-35702bd443ba5645e0c7c0f9e840918bb869dc30ff0f73665c8822d654247bff> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-03453e6e9ff15f16db131e5268955e3046c186de2581040dff34ff6d92e4cf91> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.rtl.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-8e76ea4f776bbb52b9c3f1cbfc4d43684319fa67c6e6bbc670bebcac0508925e> (4996)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.rtl.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-6de590a9097a2cb6aed95e2892646ce9fa34153ce870218eec5694cf11dc8945> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.rtl.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-ba35d1946ef145a67ea5acf3542862be195caf6413c2c8c9734c9f214ae1f04d> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.rtl.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-f954796ea6105a5046f068eec3bf93736170d036379b0f7ade93302241e42c66> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-c789c7987a1d65f2a78d931ceb8ab0e07f906a75d2f206ddf1fea22ab3602efb> (427)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-acfd604f0418ad8af497e788d920866d006f392740717aa63eaa1c5318d336c7> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-3424018c1910906ee1517a108c3d3ae2afd2d83f995b71318d49cc27dcb064db> (8)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-5a5959d3d53fa04346c209d951ad0f1278bb11dc7b885247c6bf6d1dc584cf9d> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.rtl.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-1e0b463579c09a5d10d373aaf6a43d4c911db541a683ded8e5d6606ade354ccd> (424)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.rtl.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-7fa4417798dc2be2f7aa28be8ce29d569470f12d83a51c6b91b69cf75ffe27b6> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.rtl.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-9577bd3140c950b43f62c32031e183d34c4e1a901984685c598775d8b05c1b2f> (8)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.rtl.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-4bb6042d325cdb2a57977533e871b9c8b8ad12f9982e967481044ba83bc9b7cd> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-31ad9c6fc58abe5ee075dc36a6270a1f6fbd92491f82911b2307c090cc994ad7> (4866)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-5e1e638d9f2e60be6f12f792f781b64e0a63cfb227c40bd3e9ca29041232dc46> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-d5a269e681a6a937af1c1f0d2ffb6a2257bd2016cdbb73d7e852237577f7a063> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-5adb5e676c83f17cfb75ceb7e529b70827650780463ed72d910dc156c7272adf> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.rtl.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-1393f63ba77c88eb1efe26def3524bff146d121e5e58ef1e4ac0a204afb57994> (4857)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.rtl.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-446ed211e2dd227db263895c5a2fbb493fa5060775f8433c9771c2ed99173175> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.rtl.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-968c1cf32d16b985386a3f2df96a1701dbc45dcec0f1ce942b3019644220835c> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap-utilities.rtl.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-a86034085ff58224a78b9c176cd0a495fc15939f22034c7e916aea8a99196136> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-94dd6d5eb1c353feb6a3f0f9fb29dc5ec9b2ecf6107db3baead4be4812fc74af> (11221)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-7c072bf87f2aec6690c22a407f1ede2067c0ca0e77fce470028efa02125d260f> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-5cd334806dfb97451c9f1051cb23282ef753e3ebe664f10490299ad85d7cc995> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-a145f6bc5c4037f901ae760d60064c58a99a9e73124033f41bc9ed89fa9427b8> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.rtl.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-97cc438cdfc162081001cac335a6d62b0b78ba69d1c6f4170fa6fe563035383c> (11197)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.rtl.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-0807ea6fd938c1bd33560ef1e83a9456dd597c3df9676b33506c2ffd05dda64d> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.rtl.min.css<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-8675fc5e46813bd4376df30caa4a17420d20c60e847af63d4db5be6e1e5c151b> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/css/bootstrap.rtl.min.css.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-ef3d229234e83e468c44abcad9a533057191b75da0b25292a790497cbf9e0ec2> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.bundle.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-757900631fa2df29255f37512375bacbbace8c0426cbfd869675e1d2ad4a41b1> (6780)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.bundle.js.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-a69e80cbca4e29bc01abdfaf52d8d561b014b87239082c81725dfb93fe8185c0> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.bundle.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-094ad0a30a4f67cf15cec1ad809b09818c8b8ae80258c8f199326adafda1582d> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.bundle.min.js.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-5e0e9f15f982b4a94aa14dbc3b6489edb03bf31800f8fa8afa518bd87f2c1555> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.esm.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-76127636db8eccfc9a4a49432c2af4633347779556c68ed403bbdd6f2196191f> (4977)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.esm.js.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-50d5f8d151a9af049ceda54418813667f8e032aa83673d06abea906056d2bdbe> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.esm.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-3dd58ad5ee79778c9e27513873e284aa25f78f9a05e2eab2f525cda5a5052c33> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.esm.min.js.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-0323a495b7756ff4848d5c107e3ba93abeb85b134cc3742c082d6bd0c1ba30bf> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-2d0c4d492813a37bd14386b1ba333a5eecddf317e626d836841c0cef3876a9af> (5026)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.js.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-6daa733cfc6d1ef312602dc181d64d313fd1d1dbf9c97f2d33ad8e46a81c2f63> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-29123ca98b41c360d17dc8b23ed72f2d0a736c2562b19cacf36a02e03e5e8c03> (7)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/bootstrap/dist/js/bootstrap.min.js.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-48ca4fff1e4c7b7acbdbc6efb99950c78455f74e85b7cf0ef031af4a1d6ada81> (1)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation-unobtrusive/LICENSE.txt<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-35d4299b675e756da307924faa625b10fa6c02ad1c9b5983db22e6a7fe452a96> (23)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-cc54d8d8e8f9c1d69c520ec051a67997621c6cec915e4de74ef43e52fec0a4bb> (435)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-bf69c25f903b28386af9d6e0cc0215dc5da3594c4db6f2d72154b1fdc121b869> (8)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation/LICENSE.md<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-d3014146c853eac73acc4fc9a31a87d250421ff367727f166b6f2916b6e85545> (22)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation/dist/additional-methods.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-12e2e1634909ce9c55ab25d81759ba09314af73044150c03525a5a8a63ff5b31> (1512)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation/dist/additional-methods.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-ee57435cb060d12f80414eb33c429c017d7af5c512bb1ee5cc1047983605cc45> (4)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation/dist/jquery.validate.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-66c69a109a84d40ef9be56ce5818961c88c6ab3faedc8728dfd5b9f47dcf450c> (1661)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery-validation/dist/jquery.validate.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-43d51f5b335f3740b7049ab391d2fa6462759ef0ebcd3c5fa8b808a667416138> (4)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery/LICENSE.txt<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-6ac1ac1ddf62ebb3ee5ecba604afa4a9c7e53bfe79cc9930f1419073ec0e9438> (21)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery/dist/jquery.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-6376badb22fdecad3f28c90eea3aaeb5b540c32868876a8f1c5ee5d85089467e> (10881)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery/dist/jquery.min.js<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-e54bb5351e602534045c1664a43b9eb8dfc5fd230d6a6e5af5641aa404516328> (2)
  *   A MaksimKez/HW17/hw16/hw16/wwwroot/lib/jquery/dist/jquery.min.map<https://github.com/dkobzin/tms-net15-onl/pull/78/files#diff-c7a846fe5ff1e8724930c5aa3c3d23f1235a56dc17adaf975c2832656116a636> (1)

Patch Links:

  *   https://github.com/dkobzin/tms-net15-onl/pull/78.patch
  *   https://github.com/dkobzin/tms-net15-onl/pull/78.diff

File: MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs (hunk 1 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs]: Исправил

File: MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs (hunk 2 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs]: Исправил

File: MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs (hunk 3 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Models/MeetingSettings.cs]: Исправил

File: MaksimKez/HW17/hw16/hw16/Program.cs (hunk 4 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Program.cs]: Исправил

File: MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs (hunk 5 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs]: Исправил

File: MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs (hunk 6 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs]: переделал. я все еще не понимаю в чем проблема в коде. На время "сессии" оно все отображается, но не записывается. Очень много раз прошел по коду и все еще не смог решить

Comment (dkobzin) [reply MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs]: Посмотрите в отладке как это происходит

**Comment by dkobzin** [comment]
Я думаю что вам надо поотлаживать код.

С уважением,

Дмитрий Кобзин
________________________________

File: MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs (hunk 6 above)
Comment (MaksimKez) [reply MaksimKez/HW17/hw16/hw16/Services/MeetingSettingsService.cs]: Исправлено, сама идея как исправить просто ужасна, но оно все работает и работает стабильно.

**Pull request merged**